files/templates/email/default.html

```diff
@@ -6,405 +6,109 @@
-			width: 570px;
-			margin: 0 auto;
-			padding: 0;
-			-premailer-width: 570px;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-			background-color: #cfcfcf;
-		}
-		.email-footer {
-			width: 570px;
-			margin: 0 auto;
-			padding: 0;
-			-premailer-width: 570px;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-			text-align: center;
-		}
-		.email-footer p {
-			color: #6B6E76;
-		}
-		.body-action {
-			width: 100%;
-			margin: 30px auto;
-			padding: 0;
-			-premailer-width: 100%;
-			-premailer-cellpadding: 0;
-			-premailer-cellspacing: 0;
-			text-align: center;
-		}
-		.body-sub {
-			margin-top: 25px;
-			padding-top: 25px;
-			border-top: 1px solid #E6E6E6;
-		}
-		.content-cell {
-			padding: 35px;
-		}
-		
-		@media only screen and (max-width: 600px) {
-			.email-body_inner,
-			.email-footer {
-				width: 100% !important;
-			}
-		}
-		@media (prefers-color-scheme: dark) {
-			body,
-			.email-body,
-			.email-body_inner,
-			.email-content,
-			.email-wrapper,
-			.email-masthead,
-			.email-footer {
-				background-color: #121213 !important;
-				color: #FFF !important;
-			}
-			p,
-			ul,
-			ol,
-			blockquote,
-			h1,
-			h2,
-			h3 {
-				color: #FFF !important;
-			}
-			.attributes_content,
-			.discount {
-				background-color: #222 !important;
-			}
-			.email-masthead_name {
-				text-shadow: none !important;
-			}
-		}
-		</style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+            background-color: #f5f5f5;
+        }
+        .container {
+            max-width: 800px;
+            margin: 0 auto;
+            padding: 20px;
+            background-color: #ffffff;
+            border-radius: 5px;
+            box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+        }
+        h1 {
+            text-align: center;
+            color: #333333;
+        }
+        p {
+            margin-bottom: 10px;
+            color: #333333;
+            line-height: 1.25rem;
+        }
+        .button-container {
+            text-align: center;
+            margin-top: 30px;
+            margin-bottom: 40px;
+        }
+        .button {
+            display: inline-block;
+            background-color: #1a6187;
+            color: #ffffff;
+            padding: 10px 20px;
+            text-decoration: none;
+            border-radius: 5px;
+        }
+        .raw-link {
+            display: block;
+            margin-top: 10px;
+            font-size: 14px;
+            text-align: left;
+            color: #777777;
+						word-wrap: break-word;
+        }
+        .reference-text {
+            margin-top: 20px;
+            font-size: 14px;
+            line-height: 1rem;
+        }
+        .user-info {
+            font-size: 14px;
+            color: #333333;
+        }
+        .user-info p {
+            margin-left: 20px;
+            line-height: 1rem;
+        }
+				@media (prefers-color-scheme: dark) {
```

Comment:
While we're touching the files, probably best to be consistent with spaces vs tabs. Here and L54

Reply:
just realized there's a mess in my editor with tabs and spaces, I'll fix it thanks